cmake/FlatccGenerateSources.cmake

```diff
@@ -0,0 +1,86 @@
+ cmake_minimum_required(VERSION 3.5)
```

Comment:
I would split this file in 2:
- Export the `flatcc` target as `flatcc::flatcc` with `install(TARGETS)` and `install(EXPORT)`.
- Inside `flatcc-config.cmake`, do `include("${CMAKE_CURRENT_LIST_DIR}/FlatccGenerateSources.cmake")` which then only contains the function. Instead of the `FLATCC_COMPILER`, it can then use the imported `flatcc::flatcc` executable target.

Reply:
Yes, that would be a better solution.